alchemy_news/README.md

```diff
@@ -0,0 +1,152 @@
+
+## Step-by-Step Tutorial
+
+This section outlines the steps required to deploy custom Watson AlchemyAPI News endpoints and widgets in Application Builder.
+
+### Obtain an Alchemy API Key
+
+For instructions to obtain an API key directly from the AlchemyAPI website see: [http://www.alchemyapi.com/api/register.html](http://www.alchemyapi.com/api/register.html)
+
+For instructions to obtain an API key through IBM Bluemix see: [https://console.ng.bluemix.net/catalog/alchemyapi/](https://console.ng.bluemix.net/catalog/alchemyapi/)
+
+Please note.
+
+* A Bluemix application is not required to be created or bound to the Alchemy service in Bluemix to use it.  This is ideal for basic use cases and quickly getting started.
+* All basic API keys are rate limited by requests/minute and total requests/day.
+
+
+### Configure Watson Explorer Application Builder
+
+This example includes two widgets.  both custom widgets require the use of the App Builder Endpoints feature.  If you are using a version of Watson Explorer < 11.0, you will need to install and configure the [WEX App Builder Sample Proxy](https://github.com/IBM-Watson/wex-appbuilder-sample-proxy) which is out of scope for this example.
+
+
+#### Configure Endpoints (WEX 11+)
+
+Two sample endpoints are provided, one for the text based list widget and another for the news sentiment trend widget.
+
+**News List Endpoint**
+
+This endpoint will search AlchemyData News and return summarized snippets of the top `N` news articles.
+
+1. From the Application Builder Admin console, navigate to the Endpoints section and create an Endpoint named `alchemy_news_list`.
+2. Add the following parameters with sample text to the endpoint.
+  - end
+      * now
+  - query
+      * |text=ibm|
+  - sentiment
+      * positive
+  - start
+      * now-7d
+  - max
+      * 5
+3. Copy the code from [news-list-endpoint.rb](appbuilder/alchemy-news-list-endpoint.rb)
+4. Update the API key in the endpoint to use your API key by replacing the text `YOUR_API_KEY_HERE`.
+5. Test the endpoint
+6. Save it.
+
+
+**News Sentiment Trend Endpoint**
+
+This endpoint will search AlchemyData News and return a computed histogram of news articles for a given time period so they can be graphed.
+
+1. From the Application Builder Admin console, navigate to the Endpoints section and create an Endpoint named `alchemy_news_trends`.
+2. Add the following parameters with sample text to the endpoint.
+  - end
+     * now
+  - query
+     * |text=ibm|
+  - sentiment
+     * positive
+  - start
+     * now-7d
+  - timeslice
+     * 1d
+3. Copy the code from [news-trend-endpoint.rb](appbuilder/alchemy-news-trend-endpoint.rb)
+4. Update the API key in the endpoint to use your API key by replacing the text `YOUR_API_KEY_HERE`.
+5. Test the endpoint
+6. Save it.
+
+
+
+#### Configuring the News Trend Widget
```

Comment:
It looks like there is a bit of mix up between the "list" and "trend" widgets here in the descriptions, and in the specific steps below.